### PR TITLE
Add EIP: Set upper bounds for MODEXP

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: EIPS/eip-modexp-limit.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+eip: nnnn
+title: Set upper bounds for MODEXP
+description: Each input field is restricted to a maximum of 8192 bits
+author: Alex Beregszaszi (@axic), Radoslaw Zagorowicz (@rodiazet)
+discussions-to:
```
Comment (axic) [inline EIPS/eip-modexp-limit.md:6]: Will create this once an EIP number is assigned.

File: EIPS/eip-modexp-limit.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+eip: nnnn
```
Comment (abcoathup) [inline EIPS/eip-modexp-limit.md:2]:
```suggestion
eip: 7823
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.

File: EIPS/eip-modexp-limit.md (hunk 1 above)
Comment (abcoathup) [reply EIPS/eip-modexp-limit.md:6]: Please use the template when creating a discussions topic in Eth Magicians: https://ethereum-magicians.org/c/eips/5

There isn't a need to wait for a number to be assigned.  I have been updating the titles on Eth Magicians with the number.

Comment (axic) [reply EIPS/eip-modexp-limit.md:6]: I've seen the template, but also less work to create it after the number was assigned, as no editing needed 😀

Comment (axic) [reply EIPS/eip-modexp-limit.md]: Added.

**Comment by axic** [comment]
@abcoathup please review again

**Comment by abcoathup** [comment]
> @abcoathup please review again

I'm not an EIP editor.  Need to wait for one of the editors to review.

**Comment by axic** [comment]
Ping @g11tech @lightclient @SamWilsn

File: EIPS/eip-7823.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Furthermore one could consider limiting the lengths to certain inputs only.
+
+## Backwards Compatibility
+
+This is a backwards incompatible change. Needs thorough check of transaction history to see if this would have caused failures.
+
+## Security Considerations
+
+Needs discussion.
```
Comment (g11tech) [inline EIPS/eip-7823.md:58]: could you add a `<!-- TODO -->` tag here so bot can bring this up before moving further from draft stage

**Comment by g11tech** [comment]
> Ping @g11tech @lightclient @SamWilsn

some nits else gtm

File: EIPS/eip-7823.md (hunk 3 above)
Comment (axic) [reply EIPS/eip-7823.md:62]: Added.

**Review by lightclient — APPROVED** [review]

**Pull request merged**